### DISCUSSION TITLE
License Question

### DISCUSSION THREAD

**Discussion by robwillup** [TolikPylypchuk/SharpHook#71] [Q&A] [answered]
Hi @TolikPylypchuk, great project.

I just have a question about license, please feel free to move or delete if this isn't the right place for this question.
SharpHook is released under MIT, but it uses libuiohook which is GPL v3. Would that require SharpHook to be GPL also?
Just curious because that may influence whether or not it can be used in certain contexts.

Thank you!

**Comment by TolikPylypchuk** [comment, marked as answer]
Hi! Thanks for asking!

GitHub infers libuiohook's license incorrectly - it's actually LGPL. There are two license files in the repository - [COPYING.md](https://github.com/TolikPylypchuk/libuiohook/blob/a8b65064bee6f0e409058be3e4da3811868e6da0/COPYING.md) and [COPYING.LESSER.md](https://github.com/TolikPylypchuk/libuiohook/blob/a8b65064bee6f0e409058be3e4da3811868e6da0/COPYING.LESSER.md). I guess GitHub only takes the first file into consideration.

As far as I know, linking LGPL-licensed projects to MIT-licensed projects is perfectly fine.

**Reply by robwillup** [reply]
Oh, that's right. Didn't see the other license file before.
Yes, I think that's OK. 
Thanks a lot for the quick response.
